packages/bind/spec/bindingAttributeBehaviors.ts

```diff
@@ -26,11 +26,14 @@ import { bindings as templateBindings } from '@tko/binding.template'
 import { bindings as ifBindings } from '@tko/binding.if'
 
 import '@tko/utils/helpers/jasmine-13-helper'
+import { Provider } from '@tko/provider'
```

Comment:
is this used? maybe `import type { Provider } ...`?

Reply:
That's so right, see L756.

![image](https://github.com/user-attachments/assets/15caa944-ec64-4fe3-9728-0323a4e1cb10)